aea/crypto/base.py

```diff
@@ -67,7 +68,11 @@ def public_key(self) -> str:
 
```

Comment:
note, the function signature needs updating such that `public_keys(self) -> Dict[str, str]:`